packages/js/src/constants.ts

```diff
@@ -1,8 +1,4 @@
-export const defaultOptions = {
-  urlBase: null,
-  siteId: 1,
-}
-
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
@jonkoops I deleted the `defaultOptions` object here as it no longer made sense to keep it (it'd likely cause a ton of confusion since both `urlBase` and `siteId` would be `null` or `undefined`. That made the `import/prefer-default-export` rule kick in. Would you like me to make `TRACK_TYPES` the default export or leave as is?

Reply:
Let's leave it as is. Since this file is for all types of constants we can keep the named export.